microprofile/messaging/src/test/java/io/helidon/microprofile/messaging/AsyncTestBean.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c)  2020 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.helidon.microprofile.messaging;
+
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.jupiter.api.Assertions.fail;
+
+public interface AsyncTestBean {
+    ExecutorService executor = Executors.newSingleThreadExecutor();
```

Comment:
This is a constant - naming convention wrong, also the actual behavior may be different from what you expect
- as soon as you shutdown the executor, all subsequent requests to execute anything would fail, and method `awaitShutdown` will work for a single class implementing this interface

Reply:
Fixed